.watch-status-schema.json

```
@@ -0,0 +1,14 @@
+[
+  "trigger",
+  "./",
+  {
+    "name": "schema",
+    "expression": [
+      "anyof",
+      ["match", "log/sqitch.status", "wholename"],
+      ["match", "app/package.json", "wholename"]
+    ],
+    "command": ["/usr/bin/env", "bash", "-c", "cd app && yarn run schema 1>../log/schema.stdout.log 2>../log/schema.stderr.log"],
```

`schema` was renamed to `build:schema`

```suggestion
    "command": [
      "/usr/bin/env",
      "bash",
      "-c",
      "cd app && yarn run build:schema 1>../log/schema.stdout.log 2>../log/schema.stderr.log"
    ],
```